apps/ios/Shared/Views/Database/DatabaseView.swift

```diff
@@ -67,6 +67,23 @@ struct DatabaseView: View {
     private func chatDatabaseView() -> some View {
         List {
             let stopped = m.chatRunning == false
+            Section {
+                Picker("Delete messages after", selection: $chatItemTTL) {
+                    ForEach(ChatItemTTL.values) { ttl in
+                        Text(ttl.deleteAfterText).tag(ttl)
+                    }
+                    if case .seconds = chatItemTTL {
+                        Text(chatItemTTL.deleteAfterText).tag(chatItemTTL)
+                    }
+                }
+                .frame(height: 36)
+                .disabled(m.chatDbChanged || progressIndicator)
+            } header: {
+                Text("Messages")
+            } footer: {
+                Text("This setting applies to messages in your current chat profile **\(m.currentUser?.displayName ?? "")**.")
```

Comment:
if I remember correctly some other views use chatViewName, probably should align

Reply:
I don't think it's needed in this comment, it can be long...